pkg/daemon/controller/ipinstance.go

```diff
@@ -209,8 +214,6 @@ func (r *ipInstanceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	if err := ipInstanceController.Watch(&source.Kind{Type: &networkingv1.IPInstance{}},
 		&fixedKeyHandler{key: ActionReconcileIPInstance},
-		&predicate.ResourceVersionChangedPredicate{},
```

Comment:
why remove this two predicates?

Reply:
Fixed. I had a mistake with `resource version` and `generation`.